go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/rickb777/date v1.12.5-0.20200422084442-6300e543c4d9
 	github.com/sergi/go-diff v1.2.0
 	github.com/tombuildsstuff/giovanni v0.20.0
-	github.com/tombuildsstuff/kermit v0.20230208.1135849
+	github.com/tombuildsstuff/kermit v0.20230208.1135850-0.20230209075055-34a9a084ad4b
```

Comment:
these changes are included in [v0.20230208.1135849](https://github.com/tombuildsstuff/kermit/releases/tag/v0.20230208.1135849), so we should be able to use that? The only difference in `main` at this time is [the `submodule` change](https://github.com/tombuildsstuff/kermit/compare/34a9a084ad4bdacd45bd1dc5887ddc10fb3bd898..0a7a41ccfd935c83883ec4eff998d69a21d56432)